EditorPluginExample/Resources/EditorData/Example.plugin.js

```diff
@@ -7,6 +7,7 @@ var ExamplePluginService = (function () {
         this.description = "This service demonstrates plugin functionality functionality.";
         this.serviceLocator = null;
         this.extensionWindow = null;
+        this.lastObjectName = null;
```

Comment:
Hah, I started reviewing the transpiled code... hm, we do need to sort the transpiled JS code residing along side the TS sources in the project.  This is a general issue with transpiled project code in general (editor and runtime)